plugins/frontend/backstage-plugin-github-pull-requests/src/components/PullRequestsStatsCard/PullRequestsStatsCard.test.tsx

```diff
@@ -64,7 +64,7 @@ describe('PullRequestsCard', () => {
         </EntityProvider>
       </TestApiProvider>,
     );
-    expect(await screen.findByText('2 months')).toBeInTheDocument();
+    expect(await screen.findByText('1 month, 27 days')).toBeInTheDocument();
```

Comment:
Its not perfect, but I think its good enough for an average pull request duration to include months and days.